packages/icon/test/Icon.test.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { render } from 'react-testing-library';
+import 'react-testing-library/cleanup-after-each';
+import Icon from '..';
+
+describe('Icon', () => {
+  test('should render normal size when given no size prop', () => {
+    const { getByTestId } = render(<Icon data-testid="icon" name="home" />);
+
+    const icon = getByTestId('icon');
+
+    expect(icon).toBeDefined();
+  });
+
+  test('should render given a size', () => {
+    const { container } = render(
+      <Icon data-testid="icon" name="home" size="7" />
```

Comment:
Need to update this for your recent change

Reply:
hmmm... how did this test pass given that recent change which makes the size value more literal. It would have output `icon-7` which is not `icon-5x`

Reply:
Ahh. It looks like it returns null and not undefined. Looks as if it passed.